server/mscalendar/availability_test.go

```diff
@@ -293,6 +295,30 @@ func TestReminders(t *testing.T) {
 			numReminders:   2,
 			shouldLogError: false,
 		},
+		"Remote event linked to channel in the range for the reminder. DM and channel reminders should occur.": {
+			remoteEvents: []*remote.Event{
+				{ID: "event_id", ICalUID: "event_id", Start: remote.NewDateTime(time.Now().Add(7*time.Minute).UTC(), "UTC"), End: remote.NewDateTime(time.Now().Add(45*time.Minute).UTC(), "UTC")},
+			},
+			eventMetadata: map[string]*store.EventMetadata{
+				"event_id": {
+					LinkedChannels: map[string]struct{}{"channel_id": {}},
+				},
+			},
+			numReminders:   1,
+			shouldLogError: false,
+		},
+		"Remote recurring event linked to channel in the range for the reminder. DM and channel reminders should occur.": {
+			remoteEvents: []*remote.Event{
+				{ID: "event_id_20230728", RecurringEventID: "event_id", ICalUID: "event_id", Start: remote.NewDateTime(time.Now().Add(7*time.Minute).UTC(), "UTC"), End: remote.NewDateTime(time.Now().Add(45*time.Minute).UTC(), "UTC")},
```

Comment:
Can we set the three event id fields to different values, if they are typically different values? Just to help catch issues if we change logic around using the fields. Same for the test case above